manifests/deis-router-rc.yaml

```diff
@@ -32,3 +32,15 @@ spec:
           hostPort: 2222
         - containerPort: 9090
           hostPort: 9090
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 9090
+          initialDelaySeconds: 1
+          timeoutSeconds: 1
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 9090
+          initialDelaySeconds: 1
```

Comment:
Is that enough wait time? I had to bump delay to 10 on a normal go app to just get hello world but I was going from `0` :P